### ISSUE TITLE
MLT and Flowblade's segfault

### ISSUE THREAD

**Issue by BourzRoss** [4nykey/4nykey#35]
With last flowblade release is some bug: it segfault without any errors message when start. Long time needed to find why. Thats are:
for MLT
1) Change every <sdl2> to sdl in mlt's EBUILD! 
2) Change SRC_URI="https://github.com/mltframework/mlt/archive/v${P}.tar.gz" (where P must be 6.14.0)
for Flowblade no need any changes!
After this (re)build mlt and (re)install flowblade get working prog.
Any way thank you by work off
 ebuild this application!

**Comment by 4nykey** [comment]
Yes, flowblade requires sdl1: (see [SDL_2_AND_MLT_6_6_0.md](https://github.com/jliljebl/flowblade/blob/v2.0/flowblade-trunk/docs/SDL_2_AND_MLT_6_6_0.md)), but gentoo mlt ebuild unconditionally disables sdl1 starting from https://github.com/gentoo/gentoo/commit/17888ed

**Comment by BourzRoss** [comment]
This (sdl used version) may be change on the way I wrote about. And without this change it don't work

**Comment by 4nykey** [comment]
Long story short, flowblade depends on py27 and sdl1 support. Gentoo mlt package dropped support for both recently.
https://bugs.gentoo.org/689068
https://bugs.gentoo.org/689070
